.github/workflows/.test.yml

```diff
@@ -180,6 +180,7 @@ jobs:
         with:
           directory: ./bin/testreports
           flags: ${{ matrix.codecov_flags }}
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
do we need a comment here, to outline it's only needed for uploading the results?